sdk/opa.go

```diff
@@ -477,16 +484,28 @@ type evalArgs struct {
 	instrument          bool
 }
 
-func evaluate(ctx context.Context, args evalArgs) (interface{}, ast.Value, map[string]server.BundleInfo, error) {
+func evaluate(ctx context.Context, args evalArgs) (interface{}, types.ProvenanceV1, ast.Value, map[string]server.BundleInfo, error) {
 
+	provenance := types.ProvenanceV1{
+		Version:   version.Version,
+		Vcs:       version.Vcs,
+		Timestamp: version.Timestamp,
+		Hostname:  version.Hostname,
```

Comment:
I _suspect_ that in most SDK use cases, these won't be set. Ours are set [via some magic here](https://sourcegraph.com/github.com/open-policy-agent/opa/-/blob/version/version_go1.18.go), but I'm unsure what these would look like in some Golang project that makes use of the SDK package.

Probably a problem that's out of scope for this PR, just something that came to mind when reviewing it.

Reply:
Hmm, good point. This is a bit of a copy-pasta from where Provenance structs are built up elsewhere. Version is at least valid - perhaps we should only return that for now?